Library/Formula/node.rb

```diff
@@ -25,6 +25,7 @@ class Node < Formula
   option "without-completion", "npm bash completion will not be installed"
 
   depends_on :python => :build
+  depends_on :xcode => :build
```

Comment:
Theoretically, this *should* have been necessary all along. According to both ` npm ` and the ` node ` team the ` gyp ` element of the Node install *requires* Xcode for successful building and running. I'm not sure why we haven't encountered this before, tbh.